app/services/liquid_assigns_service.rb

```diff
@@ -15,4 +15,27 @@ def assigns
 
   def assigns_pro
   end
+
+  def project_drops
+    {
+      'evidence' => project_records(type: :evidence),
```

Comment:
This will be overwritten by any `liquid_resource_assigns` coming from the controllers when it reaches `LiquidFilter` so it will have to be named differently ex 'evidences'